components/clientComponents/forms/ResumeForm/ResumeForm.tsx

```diff
@@ -17,6 +17,7 @@ import { ErrorResuming } from "./ErrorResuming";
 import { ResumeUploadIcon } from "@serverComponents/icons/ResumeUploadIcon";
 import { LightBulbIcon } from "@serverComponents/icons/LightBulbIcon";
 import { GcdsH1 } from "@serverComponents/globals/GcdsH1";
+import { useLogMessage } from "@lib/hooks/LogMessage/useLogMessage";
```

Comment:
Might want to call this useLogClientMessage ? or something so we can tell the difference as to when / where to use.

Reply:
I originally called it `useLogMessageClient` but thought it was too long. But I think you're right, especially when trying to find the right import, being specific is better.